tendrl/gluster_integration/objects/definition/gluster.yaml

```diff
@@ -60,6 +60,17 @@ namespace.gluster:
       type: Create
       uuid: b8dd2b4a-96de-4ded-b17d-374d26ec7593
       version: 1
+    SyncAggregatedClusterDetails:
+      tags:
+        - "provisioner/gluster"
```

Comment:
Should we mandate this flow to be executed just by "provisioner/gluster" node ? Can't this be picked by any gluster-integration that is free ?
I feel we can fire this job only from "provisioner/gluster" node and let the job be executed by any gluster-integration. Pls provide your thoughts